File: sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if 1 <= height <= 8:
        break

for line in range(height):
    for space in range(height - line - 1):
        print(" ", end="")
    for _ in range(line + 1):
        print("#", end="")
    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    int line;
    int column;
    int space;
    // Prompt the user for the pyramid's height
    do
    {
        height = get_int("height: ");
    }
    while (height < 1);

    // Print a pyramid of that height
    for (line = 0; line < height; line++)
    {
        // Print spaces
        for (space = 0; space < (height - line - 1); space++)
        {
            printf(" ");
        }
        // Print bricks
        for (column = 0; column <= line; column++)
        {
            printf("#");
        }
        printf("\n");
    }
}
